doc/DeveloperGuide.md

```diff
@@ -94,6 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be user-friendly, even though it has a command line interface.
+6. Should be easily portable, so as to be used on another system.
```

Comment:
Does this mean it should come as a single .jar file, or a portable installer, or the save file should be bundled together with it?